website/views/index.html

```diff
@@ -32,6 +32,25 @@
 
     <!--/.End of boostrap file link-->
     <link href="css/page.css" rel="stylesheet" />
+    <script type="text/javascript" src="js/jquery.min.js"></script>
+    <script type="text/javascript">
+    function RadioClick() {
+      var data = {};
+      data.machine = "Machine1";
+      data.status = $("input[type='radio'][name='status']:checked").val();
+
+      $.ajax({
+            type: 'POST',
+            data: JSON.stringify(data),
+                contentType: 'application/json',
+                        url: 'SendState',            
+                        success: function(data) {
+                            console.log('success');
+                            console.log(JSON.stringify(data));
+                        }
+                    });
+    }
+    </script>

```

Comment:
Use this instead: 
```
  <script>
  $(document).ready(function () {
    $(".radioButton").on("change", function () {
      data.status=this.innerText;
      //Add remaining code
    });
  });
  </script>
```  
With this, you dont have to write the tag 'onclick' for each radio button.